Assets/Scripts/Inventory/UI/UIItemSlot.cs

```diff
@@ -87,6 +105,24 @@ public void OnBeginDrag(PointerEventData eventData)
     // Required to do OnBeginDrag :/
     public void OnDrag(PointerEventData eventData) { }
 
+    public void SetItemSupplyDisplay(float value)
+    {
+        itemSupplySlider.value = value;
+        ColorBlock colors = itemSupplySlider.colors;
+        if (value > 66f)
+        {
+            colors.disabledColor = itemSupplyFullColor;
+        } else if (value > 33f)
```

Comment:
Percentages should be between 0 and 1

Reply:
Why? It is easier to round to whole numbers when using 0 to 100.